### ISSUE TITLE
Performance on segmentation 

### ISSUE THREAD

**Issue by tonysy** [erikwijmans/Pointnet2_PyTorch#23]
Hi, I wonder your performance on segmentation experiment. I only achieve ~81.0% accuracy, which is far away from the 85 % reported in the paper. Any suggestions to get similar performance with the original paper? Thanks a  lot.

**Comment by erikwijmans** [comment]
I have gotten similar results. Couple things influence it:

1) The authors never released the exact architecture they used in the paper

2) They report voxel level accuracy, while this repo does point level accuracy (voxel level accuracy tends to be higher, but is much slower to compute)

3) The default number of points is 4092 in this repo instead of the 8192 used in the paper due to memory constraints.

**Comment by tonysy** [comment]
Thanks for your reply.
Could you share the voxel level accuracy calculation script?
Also, if I use multi-GPU for training, need I adjust the hyperparameters(such as lr or decay)?

Thanks again.

**Comment by erikwijmans** [comment]
I just used the same function as Charles: https://github.com/charlesq34/pointnet2/blob/master/scannet/pc_util.py

For multi-gpu, I haven't needed to change hyper parameters.  I haven't spent much time getting the numbers to match up perfectly however.
